docs/automated-tests/test-execution/quick-run-browsers.md

```diff
@@ -1,26 +1,36 @@
 ---
 title: Browser Control in Quick Execution
 page_title: Browser Control in Quick Execution
-description: "Test Studio Quick test Execution. How to Set preferred browser for test runs and test recording. Wuick Access the browser calibration tool"
+description: "Learn how to set a preferred browser for Test Studio's quick test execution and how to calibrate a browser."
 position: 2
 ---
 # Browser Control in Quick Execution
 
-The **Test Ribbon** in Test Studio allows you to trigger a quick test execution. Along with that it provides quick access to some useful project settings.
+The **Test** ribbon in Test Studio allows you to trigger a quick test execution and provides access to project settings like preferred browser and browser calibration.
 
 ## Preferred Browser
 
-If the application under test is mainly used in one browser, or there is a requirement to test in specific browser, it would be waste of time to choose the executing browser for each test run. Therefore Test Studio allows you to set one of the supported browser as a preferred one for the current project - that way this browser will be triggered for each recording session and quick run of the test. This setting is global for the current project and can be modified in the Project Settings.
+Test Studio allows you to set a preferred browser for the current project so that you can use this browser in each test run. This is helpful in the following scenarios:
```

Comment:
Open the paragraph with the most important message that you want to convey - _you can set a preferred browser_.